include/mata/nfa.hh

```diff
@@ -732,7 +724,7 @@ public:
      * @return Returns reference element of transition list with epsilon transitions or end of transition list when
      * there are no epsilon transitions.
      */
-    Moves::const_iterator get_epsilon_transitions(State state, const Symbol epsilon = EPSILON) const;
+    Moves::const_iterator get_epsilon_transitions(State state, Symbol epsilon = EPSILON) const;
```

Comment:
why is parameter no longer constant?

Reply:
It is constant (see the function definition), but `const` in function declaration has no functionality. The compiler omits it.

Reply:
ok, legit